upload.js

```diff
@@ -17,7 +17,7 @@ async function upload(
   const url =
     BASE_URL +
     path.posix.join(
-      `/repos/${username}/${repo}/contents`,
+      `/repos/${repo}/contents`,
```

Comment:
Oh, forgot to revert that change. If you do not pass in the username but `github.repository`, you can use it in any organization as well. 

```
- uses: rajbos/github-upload-action@push-to-branch
      with:
        access-token: ${{ secrets.GITHUB_TOKEN }}
        file-path: test.txt
        username: x
        repo: ${{ github.repository }}
        remote-dir: test
        branch-name: not-existing-branch
```

Otherwise people need to know that the url is based on username/repo and then inject the organization name in the username to get things working in an org.

Reply:
Converted it to use an `owner` parameter instead of `user`.